components/style/properties/longhand/font.mako.rs

```diff
@@ -350,6 +350,58 @@ ${helpers.single_keyword("font-variant",
     }
 </%helpers:longhand>
 
+// https://www.w3.org/TR/css-fonts-3/#font-size-adjust-prop
+// FIXME: This prop should be animatable
+<%helpers:longhand products="none" name="font-size-adjust" animatable="False">
+    use values::CSSFloat;
+    use values::NoViewportPercentage;
+    use values::computed::ComputedValueAsSpecified;
+
+    impl ComputedValueAsSpecified for SpecifiedValue {}
+    impl NoViewportPercentage for SpecifiedValue {}
+
+    #[derive(Clone, Debug, PartialEq)]
+    #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+    pub enum SpecifiedValue {
+        None,
+        Number(CSSFloat),
```

Comment:
You can use [`Number`](https://github.com/servo/servo/blob/f48b3fe2197a59b29bc711fb1b5496cbb97bd330/components/style/values/specified/mod.rs#L466) struct here (which stores number as CSSFloat but it has useful implementations). And in parsing function, you can try to parse number directly with:
`SpecifiedValue::Number(try!(Number::parse_non_negative(input)))`